js/weather.js

```diff
@@ -0,0 +1,219 @@
+/* function insertWeatherData (x) {
+    // creates article element to display weather data
+    var elementToAdd = document.createElement("article");
+    elementToAdd.className = "message";
+    elementToAdd.classList.add("is-primary");
+    elementToAdd.classList.add("is-large")
+
+    // creates message element in article
+    var child = document.createElement("div");
+    child.className = "message-body";
+
+    // create texts and appends it to the message element created
+    var node = document.createTextNode(x);
+    child.appendChild(node);
+
+    // appends message to article
+    elementToAdd.appendChild(child);
+    elementToAdd.id = "weatherObject";
+
+    // creates a button
+    var celsius = document.createElement("button");
+    celsius.className = "button";
+    celsius.classList.add("is-primary");
+    celsius.classList.add("is-outlined");
+    celsius.textContent = "C";
+    celsius.id = "celsius";
+
+    var farenheit = document.createElement("button");
+    farenheit.className = "button";
+    farenheit.classList.add("is-primary");
+    farenheit.classList.add("is-outlined");
+    farenheit.textContent = "F";
+    farenheit.id = "fahrenheit";
+
+    // creates new row
+    var newDiv = document.createElement('div');
+    newDiv.appendChild(celsius);
+    newDiv.appendChild(farenheit);
+    
+    // appends article to container
+    document.getElementById("contain").appendChild(elementToAdd);
+    document.getElementById("contain").appendChild(newDiv);
+
+
+} */
+
+function insertWeatherData (x, weather) {
```

Comment:
I suppose `x` is the temperature. Please rename it for clarity.

Suggestion for `weather`: 

- `condition`
- `weatherCondition`